index.js

```diff
@@ -15,7 +15,9 @@ let screenHeight = Dimensions.get('window').height;
  */
 const widthPercentageToDP = widthPercent => {
   // Parse string percentage input and convert it to number.
-  const elemWidth = parseFloat(widthPercent);
+  const elemWidth = parseFloat(
+    typeof widthPercent === "number" ? widthPercent : widthPercent.toString()
```

Comment:
why not do it 
```
const elemWidth = typeof widthPercent === "number" ? parseFloat(widthPercent)
```

Reply:
You mean `const elemWidth = typeof widthPercent === "number" ? widthPercent : parseFloat(widthPercent)`?